src/Symfony/Component/Workflow/DataCollector/WorkflowDataCollector.php

```diff
@@ -88,33 +88,46 @@ public function getCallsCount(): int
         return $i;
     }
 
+    public function hash(string $string): string
+    {
+        return hash('xxh128', $string);
+    }
+
+    public function buildMermaidLink(string $name): string
```

Comment:
```suggestion
    public function buildMermaidLiveLink(string $name): string
```

Maybe? As Mermaid is the format

Reply:
but it's mermaid_link. Anyway, PR updated. thanks